docs/source/examples/demo/chicago_bike/__init__.py

```diff
@@ -0,0 +1,13 @@
+from demo import PWD
+from pandas import read_csv
+from os.path import join
+
+PWD = join(PWD, 'chicago_bike')
+
+
+def load_sample():
+    return read_csv(
+        join(PWD, 'sample.csv'),
+        parse_dates=['starttime', 'stoptime'],
+        index_col='trip_id',
+    )
```

Comment:
Super minor detail, but add newline at end of file per lint convention - I don't think this is being checked by the linter